tests/unit/mocks.py

```diff
@@ -56,8 +59,8 @@ class PCEObjectMock(object):
     """
     Base class for PCE object mocks
     """
-    base_pattern = '^/api/v2(?:/orgs/\d+)?(?:/sec_policy/(?:draft|active))?/([a-zA-Z_\-]+)'
-    href_pattern = '^/api/v2((?:/orgs/\d+)?(?:/sec_policy/(?:draft|active))?(?:/[a-zA-Z_\-]+/[a-zA-Z0-9\-]+)+)$'
+    base_pattern = r'^/api/v2(?:/orgs/\d+)?(?:/sec_policy/(?:draft|active))?/([a-zA-Z_\-]+)'
```

Comment:
Again, not sure why these have been changed as they were working fine before.

Reply:
Here we have redundant escape characters as well as PEP8 W605 Invalid escape sequences, converting to raw string literal helps avoid these issues.